content/administration/upgrade.rst

```diff
@@ -1,264 +1,314 @@
-  installed and the respective subscription is still active; and
-- the upgrade of all **developments and customizations covered by a maintenance of customizations
-  subscription**.
+Which version of Odoo should I upgrade to ?
+===========================================
 
-Upgrade services are limited to the technical conversion and adaptation of a database (standard
-modules and data) to make it compatible with the version targeted by the upgrade.
+We always recommend upgrading **to the latest version of Odoo** to benefit from the latest features, improvements, and security fixes. If you are hosted on Odoo Online, you will automatically be enrolled in the rolling release program and will always be on the latest version of Odoo if your database can be upgraded automatically.
 
-Upgrade services not covered by the SLA
----------------------------------------
+.. _upgrade_faq/rolling_release:
 
-The following upgrade-related services are **not** included:
+What is the Rolling Release (applicable to Odoo Online databases) ?
+===================================================================
 
-- the **cleaning** of pre-existing data and configurations while upgrading;
-- the upgrade of **custom modules created in-house or by third parties**, including Odoo partners;
-- lines of **code added to standard modules**, i.e., customizations created outside the Studio app,
-  code entered manually, and :ref:`automated actions using Python code
-  <studio/automated-actions/action>`; and
-- **training** on using the upgraded version's features and workflows.
 
-.. note:: |assistance-contact|
+This feature allows customers to upgrade their database directly from a message prompt sent to the database administrator as soon as the new version is released. Odoo first tests the upgrade to the next version and if no issues are detected, the database administrator receives a message with 2 options:
 
-.. seealso::
-   - :doc:`Upgrade FAQ <upgrade/faq>`
-   - :doc:`Odoo.sh documentation <odoo_sh>`
-   - :doc:`Supported Odoo versions <maintain/supported_versions>`
+#. To 'Upgrade Now', which immediately triggers the upgrade of your live production database. (Discouraged)
+
+#. To take you to your `database manager <https://www.odoo.com/my/databases/>`_ where you can `request an upgraded test database <https://upgrade.odoo.com/#online/>`_ and check the upgraded test database for any discrepancies. (Recommended)
+
+If a database cannot be successfully upgraded automatically, then a ticket is automatically created to allow the upgrade technicians of Odoo to investigate the issue and fix it. Once the issue is fixed, the database is unblocked from the process and is available to be tested automatically again
+
+.. note::
+    In case of a database running a version that will soon be unsupported, the database can be automatically upgraded to the latest supported version of Odoo through the rolling release process.
```

Comment:
```suggestion
    If a database runs a version that will soon be unsupported, the database can be automatically upgraded to the latest supported version of Odoo through the rolling release process.
```
Do we have Odoo Online DBs running Odoo 14? If it's an unlikely/impossible scenario, we might want to remove that note.

Reply:
Yes, databases for which there is an issue with their upgrade (deactivated view, traceback during the upgrade process, ...)  I took this screenshot on a SaaS database currently running Odoo 14 and wishing to upgrade 
![image](https://github.com/odoo/documentation/assets/32837941/d954f983-a32b-4b35-85f6-404f86475d60)